coordinator/core/clientapi.go

```diff
@@ -296,6 +311,15 @@ func (c *Core) UpdateManifest(ctx context.Context, rawUpdateManifest []byte, upd
 		return err
 	}
 
+	oldLog, err := c.data.getUpdateLog()
+	if err != nil {
+		return err
+	}
+	c.updatelog.reset(oldLog)
+	for pkgName, pkg := range updateManifest.Packages {
+		c.updatelog.logger.Info("SecurityVersion increased", zap.String("user", updater.Name()), zap.String("package", pkgName), zap.Uint("new version", *pkg.SecurityVersion))
```

Comment:
That seems cumbersome. I think the updateLog code should be in its own package and be usable like this:
```go
updateLogger := updateLog.New()
updateLogger.Info(...) // should work by embedding zap.Logger into the updateLogger struct
txdata.appendUpdateLog(updateLogger.String())
```